apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -121,6 +121,26 @@ defmodule OMG.Watcher.ExitProcessor.Core do
      }}
   end
 
+  @doc """
+  Use to check if the settings regarding the `:exit_processor_sla_margin` config of `:omg_watcher` are OK.
```

Comment:
I like your PR description and don't want it to fade away with the closed PR. Can we add it here?

```elixir
@doc """
Check if the settings regarding the `:exit_processor_sla_margin` config of `:omg_watcher` are OK.

Since there are combinations of our configuration that may lead to a dangerous setup of the Watcher
(in particular - muting the reports of unchallenged_exits), we're enforcing that the exit_processor_sla_margin
be not larger than min_exit_period.

This check can be overridden (that is, allowing sla_margin to be larger than min_exit_period) when one know's what one's doing 
(e.g. in tests, dev environments, where the min_exit_period is very short), by passing `true` to `sla_margin_force`.
"""

```

Reply:
:+1: taking the liberty to distribute the description to this here (the "Since..." part) and `details.md` other docs (the "can be overriden" part)